app/src/notifications/components/notifications.scss

```diff
@@ -43,3 +38,47 @@
     }
   }
 }
+
+.notification {
+  background-color: $color-19;
```

Comment:
@j8seangel @satellitestudiodesign shouldn't the background of the notification have less opacity? It feels weird to have such a solid color, at least based on the  image included in the PR description.

Reply:
@enriquetuya We included the differentiation between notifications, warnings, and errors precisely to grab users attention when reading the message is mandatory. That's why I went with solid, strong color blocks.